logging-test/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+plugins {
+    kotlin("multiplatform")
+    id("kotlinx-atomicfu")
+    id("org.jmailen.kotlinter")
+    id("com.vanniktech.maven.publish")
```

Comment:
Is this module just for internal testing facilities, or did you purposely have this configured for publication?

Reply:
Just something I missed when I was stripping this gradle file down. I'll remove.